site/templates/film/item.jet

```diff
@@ -62,94 +80,106 @@
           {{end}}
 
           {{yield awardNominations() film}}
-          <s72-element-switcher>
-            {{if isEnabled("element_switcher_enabled")}}
-              <div class="meta-detail-switcher-tagline">
-                <p>{{film.Tagline}}</p>
-              </div>
-            {{end}}
-            <div class="{{ isEnabled("element_switcher_enabled") ? "s72-hide" : "" }}"> {* hide class prevents FOUC *}
-              <div class="meta-detail-synopsis">{{film.Overview | raw}}</div>
-              <div class="meta-detail-cast">
-              {{if len(film.Cast) > 0 }}
-                <h2>{{i18n("meta_detail_cast_title")}}</h2>
+          <div class="element-switcher-sponsor">
```

Comment:
This falls in the same boat as the previous one, perhaps I need to get over myself.

```suggestion
          <div class="element-switcher-wrapper">
```

Reply:
- changed